float64 pitch_x
float64 roll_y
float64 yaw_z